src/OpenTelemetry.Exporter.Console/OpenTelemetry.Exporter.Console.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <Authors>OpenTelemetry authors</Authors>
+    <PackageIconUrl>https://opentelemetry.io/img/logos/opentelemetry-icon-color.png</PackageIconUrl>

```

Comment:
`PackageIconUrl` is deprecated.

```suggestion
```

`PackageIcon` is also set at Common.prod.props level:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/cb0630c8f9aeb0b937f05538420867d9d9e5e37d/build/Common.prod.props#L31